hack/verify-gofmt.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script checks whether the source code needs to be formatted or not by
+# `gofmt`. Run `hack/update-gofmt.sh` to actually format sources.
```

Comment:
What about bring some notes in the README, and clarify the difference between both.

Reply:
Sure Amim added to README